.gitignore

```diff
@@ -1,9 +1,9 @@
-*.gem
+.ruby-gemset
+.ruby-version
 *.rbc
 .bundle
 .config
 .yardoc
-Gemfile.lock
```

Comment:
I removed it from .gitignore and added it to the repo. IMO in most cases the Gemfile.lock should be checked in. Only this makes builds repeatable. There is an argument to be made whether the lock file should not be checked in got gem development, but I prefer to lock the dependencies down ensuring anyone who wants to build the source can do so with the right dependencies. Verifying potential dependency changes can then be done by a dedicated person or at dedicated times where one also can then address potential incompatibilities. See also https://viget.com/extend/bundler-best-practices

Bottom line, I think `Gemfile.lock` belongs under version control. This is, however, independent of this issue for course. If the majority of people is against it, I am fine changing this. Hence I added it as last commit. I could also imagine creating a dedicated issue and pull request to separate this out.